README.md

```diff
@@ -34,6 +34,7 @@
 * [Custom Directives](#custom-directives)
 * [Dialog](#dialog)
 * [Error Handling](#error-handling)
+* [Show Support](#show-support)
```

Comment:
It's an "Echo Show" right? I would say that.

Reply:
👍